### PR TITLE
Small nits

### PR DESCRIPTION
**Pull request by 0x0Louis**
Small fix for `becomeOwner`, added tests and typos

### CODE REVIEW

File: contracts/utils/PausableAdmin.sol [hunk 1]
```diff
@@ -133,13 +159,4 @@ contract PausableAdmin is PendingOwnable, Pausable, IPausableAdmin {
...
     }
-
-    /**
-     * @notice Internal function to revoke the role of an admin
-     */
-    function _revokePauseAdmin(address _revokedAdmin) internal {
-        if (!_pauseAdmins.remove(_revokedAdmin))
-            revert PausableAdmin__AddressIsNotPauseAdmin(_revokedAdmin);
-        emit PauseAdminRevoked(_revokedAdmin);
-    }
```
Comment (0x0Louis) [inline contracts/utils/PausableAdmin.sol:144]: Removed this part as OZ does it that way (https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/AccessControl.sol#L179-L183)

File: contracts/utils/PausableAdmin.sol [hunk 2]
```diff
@@ -70,6 +79,23 @@ contract PausableAdmin is PendingOwnable, Pausable, IPausableAdmin {
             super.supportsInterface(interfaceId);
     }
 
+    /**
+     * @notice Transfers the ownership to the new owner (`pendingOwner).
```
Comment (0xnivek) [inline contracts/utils/PausableAdmin.sol:83]: nit: think you're missing a trialing ` after "pendingOwner"

File: contracts/utils/PausableAdmin.sol [hunk 3]
```diff
@@ -50,6 +50,15 @@ contract PausableAdmin is PendingOwnable, Pausable, IPausableAdmin {
         return _pauseAdmins.length();
     }
 
+    /**
+     * @notice View function to check whether an user is an admin (true) or not (false)
```
Comment (0xnivek) [inline contracts/utils/PausableAdmin.sol:54]: nit: `whether a user`

**Review by 0xnivek — APPROVED** [review]
looks great!

**Pull request merged**